api/api-rules/violation_exceptions.list

```diff
@@ -532,6 +537,7 @@ API rule violation: names_match,k8s.io/api/core/v1,RBDVolumeSource,RBDPool
 API rule violation: names_match,k8s.io/api/core/v1,RBDVolumeSource,RadosUser
 API rule violation: names_match,k8s.io/api/core/v1,VolumeSource,CephFS
 API rule violation: names_match,k8s.io/api/core/v1,VolumeSource,StorageOS
+API rule violation: names_match,k8s.io/api/networking/v1,IngressSpec,DefaultBackend
```

Comment:
this is unexpected... this is actually flagging that the json tag doesn't match the renamed field, which should be fixed; see https://github.com/kubernetes/kubernetes/pull/89778/files#r405059319